metadata.yaml

```diff
@@ -12,7 +12,7 @@ resources:
     type: oci-image
     description: 'Backing OCI image'
     auto-fetch: true
-    upstream-source: docker.io/kubeflownotebookswg/centraldashboard:v1.8.0
+    upstream-source: docker.io/kubeflownotebookswg/centraldashboard:v1.9.0-rc.0
```

Comment:
Hasn't upstream released a newer image than `rc.0`?

Reply:
looks like they [have](https://github.com/kubeflow/manifests/blob/master/apps/centraldashboard/upstream/base/kustomization.yaml#L16) but it's only in `master` not `1.9.0-rc.2`

Reply:
I've updated it